subsystem-test/tests/src/test/java/org/jboss/as/subsystem/test/otherservices/OtherServicesSubsystemTestCase.java

```diff
@@ -88,7 +88,7 @@ public void testOtherService() throws Exception {
 
         Assert.assertNotNull(services.getContainer().getService(OtherService.NAME));
         Assert.assertNotNull(services.getContainer().getService(MyService.NAME));
-        MyService myService = (MyService)services.getContainer().getService(MyService.NAME).getValue();
+        MyService myService = (MyService)services.getContainer().getService(MyService.NAME).awaitValue();
```

Comment:
@ropalka Wouldn't we need to add a timeout to the wait condition? I'm just wondering if that can prevent any kind of hang running the test suites.

We can also proceed with this as it is and react in case we get these tests hanging the whole test suite. WDYT?

Reply:
It is not needed @yersan . There is no async involved in these scenarios so it shouldn't result into hang.